include/fmt/std.h

```diff
@@ -435,6 +437,28 @@ struct formatter<BitRef, Char,
   }
 };
 
-FMT_END_NAMESPACE
+FMT_EXPORT
+template <typename T, typename Char>
+struct formatter<std::atomic<T>, Char> : formatter<T, Char> {
+  template <typename FormatContext>
+  auto format(const std::atomic<T>& v, FormatContext& ctx) const
+      -> decltype(ctx.out()) {
+    return formatter<T, Char>::format(v.load(), ctx);
+  }
+};
 
+FMT_EXPORT
+template <typename Char>
+struct formatter<std::errc, Char>
+    : formatter<std::underlying_type<std::errc>::type, Char> {
+  template <typename FormatContext>
+  FMT_CONSTEXPR auto format(std::errc v, FormatContext& ctx) const
+      -> decltype(ctx.out()) {
+    using underlying_type = std::underlying_type<std::errc>::type;
+    return formatter<underlying_type, Char>::format(
+        static_cast<underlying_type>(v), ctx);
+  }
+};
```

Comment:
I am not sure if providing a formatter for errc is a good idea. I think it should be up to the user to decide how they want to present it and if they want a number they could use to_underlying.

Reply:
Is there any way to provide a default formatter if no other formatter exists?

Reply:
Considering that it is easy to format `errc` as a number with `to_underlying`, I don't think it's worth exploring the default formatter option.